### ISSUE TITLE
How does the user initially populate their collections?

### ISSUE THREAD

**Issue by aeschylus** [sul-dlss-deprecated/mirador_sul#14]
Is it only drag and drop + url pasting? Where does this happen? How do they make collections from items that are already somewhere in another collection on the same account?

**Comment by cncoleman** [comment]
**1. From the workspace:**  Adding objects to the workspace from external sources or from the user library creates a default collection for that workspace. 
**2. From the library:** Select objects in your library with options "Create collection from selected", "Add selected to existing collection", "Create new workspace from selection"
**3. From an existing collection:** Select objects and same options appear as with library, "Create collection from selected", "Add selected to existing collection", "Create new workspace from selection"

**Comment by csbailey5t** [comment]
@cncoleman Can a user actually create a workspace without already establishing a collection? My impression from the technical meeting was that a workspace is attached to a collection, i.e., you can only create a workspace from an existing collection. If that is the case, you could create an empty collection, then create a workspace and populate a collection through the workspace.

**Comment by cncoleman** [comment]
@csbailey5t Hopefully your question on 2. will be answered in the mockups I posted to Slack today. 

RE: 1. I'm not aware of the technical constraints around having to start with collections to get to a workspace. If that's the case, we would need to drop 1.  But if we can create a collection by populating a workspace without technical complication, I think it would be a useful additional workflow. 

There are often needs for low risk, low commitment uses that favor 'jumping right in'. If I'm creating a collection from my own selected images, why should I need to bother with any pre-existing collections? And If I don't need to have a pre-existing collection, it would be cumbersome to start by creating a collection before I get to use the viewer.

**Comment by blalbrit** [comment]
@csbailey5t @cncoleman 👍 
re: 1 - can a user actually create a workspace without already establishing a collection?
* Answer 1 - no, a workspace must be attached to a collection
* Answer 2 - a user could start with a blank collection to which a workspace is attached
* Answer 3 - Nicole's "it would be cumbersome to start by creating a collection..." suggests 2 is necessary

re: 2 - we had discussed, and I am in favor of, not exposing the Library at this stage. It is simply a data model at this point that stands in for the User - and we did not want to expose the User ID at any point in the process, so the Library is proxy for User in practical terms. If, in the future, working with the Library as a manipulable set of materials... we can discuss further. Seems out of scope for May.
